# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/looking-glass/hotfix-module-install.patch
# Copyright (C) 2026 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- LookingGlass.git/module/Makefile.vanilla
+++ LookingGlass.git/module/Makefile
@@ -19,4 +19,8 @@
 	sudo chown $(USER) /dev/uio0
 	sudo chown $(USER) /dev/kvmfr0
 
+module-install:
+	$(MAKE) -C $(KDIR) M=$(PWD);
+	$(MAKE) -C $(KDIR) M=$(PWD) modules_install;
+
 .PHONY: test
